lms/djangoapps/ccx/views.py

```diff
@@ -398,8 +399,14 @@ def ccx_student_management(request, course):
             unenroll_email(ccx, email, email_students=False)
     except ValidationError:
         log.info('Invalid user name or email when trying to enroll student: %s', email)
+        if email:
+            error_message = ('Could not find a user with name or email'
+                             ' "{}" '.format(email))
```

Comment:
The way the line is broken is what I intended to point out; it should be formatted like one of the following:
```python
        error_message = (
            'Could not find a user with name or email "{}"'.format(email)
        )
```
```python
        error_message = 'Could not find a user with name or email "{}"'.format(
            email
        )
```
As the example in this section shows: https://github.com/edx/edx-platform/wiki/Python-Guidelines#breaking-long-lines